code/modules/mob/living/carbon/carbon.dm

```diff
@@ -767,28 +767,33 @@
 	else
 		hud_used.healths.icon_state = "health6"
 
-/mob/living/carbon/update_stamina_hud(shown_stamina_amount)
+/mob/living/carbon/update_stamina_hud(shown_stamloss)
```

Comment:
Why was the variable name changed to be worse?

Reply:
because stamina amount =/= stamina lost

Reply:
I see, should be `shown_stamina_loss` and changed across all definitions then

Reply:
👍